index.html

```diff
@@ -1,11 +1,132 @@
-<html>
+ <!DOCTYPE html>
+<html lang="en">
 <head>
-<meta http-equiv="Refresh" content="0; url=https://borgbackup.readthedocs.io/" />
+	<meta charset="utf-8"/>
+	<title>BorgBackup – Deduplicating archiver with compression and authenticated encryption</title>
+	<style type="text/css" id="internal-style">
+		body {
+			margin: 0;
+			font-family: verdana,arial,helvetica,sans-serif;
+		}
+
+		header {
+			background-color: black;
+			padding-top: 4em;
+			padding-bottom: 1em;
+			border-bottom: 2px solid #00dd00;
+		}
+
+		#logo {
+			width: 600px;
+			margin: 0 auto;
+		}
+
+		h1 {
+			margin: 0;
+			margin-left: -20px;
+		}
+
+		main {
+			width: 600px;
+			margin: 0 auto;
+		}
+
+		h2 {
+			font-size: 220%;
+			font-weight: normal;
+			padding-bottom: .7em;
+			border-bottom: 1px solid #666;
+		}
+
+		main ul {
+			line-height: 1.5em;
+		}
+
+		#links {
+			margin-top: 2em;
+		}
+
+		#links a {
+			text-decoration: none; color: #00dd00;
+		}
+
+		#links a:hover .action {
+			text-decoration: none;
+			font-weight: bold;
+			color: #00ff00;
+			background: white;
+		}
+
+		#links .link {
+			float: left;
+			width: 199px;
+			height: 120px;
+			text-align: center;
+		}
+
+		#links .action {
+			font-size: 400%;
+		}
+	</style>
+	<link rel="shortcut icon" href="favicon.ico"/>
+	<link rel="icon" type="image/ico" href="favicon.ico"/>
 </head>
 <body>
-<p>
-<a href="https://borgbackup.readthedocs.io/">Borg Backup</a>
-<a href="https://borgweb.readthedocs.io/">Borg Web</a>
-</p>
+
+	<header>
+		<div id="logo">
+			<h1><a href="/"><img src="logo.svg" alt="Borg logo"></a></h1>
+		</div>
+	</header>
+
+	<main>
+		<h2 id="tagline">Deduplicating archiver<br>with compression and encryption</h2>
+
+		Borg Backup (short: Borg) gives you:
+
+		<ul>
+			<li>Space efficient storage of backups.
+			<li>Secure, authenticated data encryption.
```

Comment:
"data encryption" could be read as "only data, not metadata, encryption".

so leave away "data"? or say "data and metadata"?